README.md

```diff
@@ -627,10 +627,73 @@ This example updates a cluster to use a new profile `my-new-profile`:
 ```
 arlon cluster update eks-1 --profile my-new-profile
 ```
-# Implementation details
 
+# Next-Generation (gen2) Clusters - New in version 0.9.x
+
+Gen1 clusters are limited in capability by the Helm chart used to deploy the infrastructure resources.
+Advanced Cluster API configurations, such as those using multiple MachinePools, each with different
+instance types, is not supported.
+
+Gen2 clusters solve this problem by allowing you to create workload clusters from a *base cluster*
+that you design and provide in the form of a manifest file stored in a git directory. The manifest
+typically contains multiple related resources that together define an arbitrarily complex cluster.
+If you make subsequent changes to the base cluster, workload clusters originally created from it
+will automatically acquire the changes.
+
+Before a manifest directory can be used as a base cluster, it must first be "prepared" or "prepped"
+by Arlon. The "prep" phase makes minor changes to the directory and manifest to help Arlon deploy
+multiple copies of the cluster without naming conflicts.
+
+To determine if a git directory is eligible to serve as base cluster, run the `basecluster validategit` command:
+```
+arlon basecluster validategit --repo-url <repoUrl> --repo-path <pathToDirectory> [--repo-revision revision]  
```

Comment:
We should also add `--repo-alias` examples. I can work on a dedicated section for `repo-alias`, and update existing examples.

Reply:
Thanks, forgot about that. Since you're the expert on this topic, please add this info in a separate PR. Thanks!